tests/accessor/accessor_exceptions_core.cpp

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for accessor exceptions.
+//
+//  This test provides verifications that exception really has been thrown for
+//  generic accessor, host_accessor and local_accessor with generic types.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "../common/disabled_for_test_case.h"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !defined(__HIPSYCL__) && !defined(__COMPUTECPP__) && \
+    !defined(__SYCL_COMPILER_VERSION)
+
+#include "accessor_exceptions.h"
+
+using namespace accessor_exceptions_test;
+using namespace accessor_tests_common;
+#endif
+
+namespace accessor_exceptions_test_core {
+using namespace sycl_cts;
+
+DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
+("Generic sycl::accessor constructor exceptions test core types.",
+ "[accessor]")({
+#ifndef SYCL_CTS_ENABLE_FULL_CONFORMANCE
+  const auto types = get_lightweight_type_pack();
+#else
+  const auto types = get_full_conformance_type_pack();
+#endif  // SYCL_CTS_ENABLE_FULL_CONFORMANCE
```

Comment:
Is there a reason to have all these instead of having a `get_conformance_type_pack()` doing the dispatch to 1 of each?

Reply:
`get_conformance_type_pack` has been added in this commit: 211016b8645e6111b46b8186b6f9153e7baa4739
Also, the same cases has been replaced into `get_conformance_type_pack` call for all these cases for tests on accessor